_style/method-invocation.md

```diff
@@ -92,25 +92,34 @@ gray area is short, operator-like methods like `max`, especially if commutative:
     // fairly common
     a max b
 
-Symbolic methods which take more than one parameter (they do exist!)
-may still be invoked using infix notation, delimited by spaces:
+Symbolic methods which take more than one parameter are discouraged.
+When they exist, they may still be invoked using infix notation, delimited by spaces:
 
     foo ** (bar, baz)
 
 Such methods are fairly rare, however, and should normally be avoided during API
-design. For example, the use of the `/:` and `:\` methods should be avoided in
+design. For example, the use of the (now deprecated) `/:` and `:\` methods should be avoided in
 preference to their better-known names, `foldLeft` and `foldRight`.
 
 ### Higher-Order Functions
 
-As noted, methods which take functions as parameters (such as `map` or
-`foreach`) should be invoked using infix notation. It is also *possible* to
-invoke such methods in the following way:
+Invoking higher-order functions may use parens or braces, but in
+either case, use dot notation:
+
+    names.map{ _.toUpperCase }
+
+Not infix:
+
+    // wrong!
+    names map { _.toUpperCase }
```

Comment:
Just below this you have:
```scala
    // right!
    names map { _.toUpperCase } filter { _.length > 5 }
```

At this point I suggest deleting the whole section since it confuses rather than enlightens.

Reply:
Oops, good catch! I've pushed a fix.

I don't know about deleting the whole section, but my fix does shorten it considerably.